device.go

```diff
@@ -187,6 +187,15 @@ func makeBlockDevIDForHypervisor(deviceID string) string {
 }
 
 func (device *BlockDevice) attach(h hypervisor, c *Container) (err error) {
```

Comment:
@devimc @amshinde are we doing this only for block devices. Can we do this for network devices also?

Reply:
This PR only intends to fix https://github.com/clearcontainers/runtime/issues/802
We have discussed that @devimc is gonna submit a more global PR for removing all the coldplug codepaths (see my comment on #471). This PR will be blocked until we move to qemu2.9

Reply:
fair enough.